cmd/client.go

```diff
@@ -9,13 +9,23 @@ import (
 	"knative.dev/func/pkg/builders/buildpacks"
 	"knative.dev/func/pkg/config"
 	"knative.dev/func/pkg/creds"
+	k8sdeployer "knative.dev/func/pkg/deployer/k8s"
+	knativedeployer "knative.dev/func/pkg/deployer/knative"
+	"knative.dev/func/pkg/describer"
+	k8sdescriber "knative.dev/func/pkg/describer/k8s"
+	knativedescriber "knative.dev/func/pkg/describer/knative"
 	"knative.dev/func/pkg/docker"
 	fn "knative.dev/func/pkg/functions"
 	fnhttp "knative.dev/func/pkg/http"
 	"knative.dev/func/pkg/k8s"
-	"knative.dev/func/pkg/knative"
+	"knative.dev/func/pkg/lister"
+	k8slister "knative.dev/func/pkg/lister/k8s"
```

Comment:
Let's restructure these packages to be based on dependency name first, type second (I'll explain in further detail via IM)